imjoy/imjoyPluginEngine.py

```diff
@@ -26,6 +26,8 @@
 # import webbrowser
 from aiohttp import streamer, web
 
+from imjoyUtils import get_psutil
```

Comment:
@MartinHjelmare I just noticed that you are placing `get_psutil` inside `imjoyUtils.py`, maybe we should not, because `imjoyUtils` are only used by the plugin workers (plugin processes) right now. During runtime, the engine and plugin workers are running in different process and virtual env, in the future, they will even on different node (e.g. with slurm). I think we should clearly separate the two, even that means we will create a copy.

Reply:
Yes.